### DISCUSSION TITLE
Xp Number

### DISCUSSION THREAD

**Discussion by Elex000** [theomenden/TheOmenDen.Vanadium#61] [Q&A]
I just tried to color my Xp Number with the Optifine color.properties but it didn't work.
Is it implemented or do I need to change something?

**Comment by DROEFTOETERX** [comment]
Didn't work for me either. I used this website to generate a pack to do it: https://enchanted.games/generator/custom-xp-colours/. For the XP counter, it will create some files in assets/minecraft/shaders/core that you have to put in your own resource pack.
